docs/deployment/docker-compose.mdx

```diff
@@ -46,16 +47,34 @@ To load sample data, copy files from the topaz repo into the mounted /data direc
 
 ```sh
 cd topaz/docs/deployments/docker-compose
-cp ../../../assets/citadel/citadel_objects.json data
-cp ../../../assets/citadel/citadel_relations.json data
-cp ../../../assets/todo/manifest.yaml data
-cp ../../../assets/todo/todo_objects.yaml data
-cp ../../../assets/todo/todo_relations.yaml data
+mkdir data
+cp ../../../assets/citadel/citadel_objects.json data/
+cp ../../../assets/citadel/citadel_relations.json data/
+cp ../../../assets/todo/manifest.yaml data/
+cp ../../../assets/todo/todo_objects.json data/
+cp ../../../assets/todo/todo_relations.json data/
 ```
 
-Now invoke the topaz CLI inside the running container to import the sample manifest and data into the topaz edge directory.
+While topaz is running, from another terminal invoke the topaz CLI inside the running container to import the sample manifest and data into the topaz edge directory.
 
 ```sh
+cd topaz/docs/deployments/docker-compose
 docker compose exec topaz ./topaz manifest set --no-check -i /data/manifest.yaml
```

Comment:
Unfortunately this doesn't work because `topaz templates install todo` does arguably more than advertised, it also rewrites the config file and attempts to contact a docker service to start a topaz container. 

```
> docker compose exec topaz ./topaz templates install todo

Installing this template will completely reset your topaz configuration.
Do you want to continue? (y/N) y
topaz: error: failed to run "docker ps -q -f name=topaz: exec: "docker": executable file not found in $PATH"
```